README.md

```diff
@@ -60,11 +60,6 @@ The leaderboard website displays scores submitted by different players. It also
 
 - User-friendly Interface
 
-## 🚀 Live Demo <a name="live-demo"></a>
-
-- [Coming Soon](#)
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- GETTING STARTED -->
```

Comment:
- [ ] [optional] @GhausAliKhan great work implementing the change requested however you do not have a link to the MIT license, it would be nice if you had one for individuals wanting to use your code. Also, it would be nice to add how to run the project locally eg command npm start.